catalystwan/api/templates/models/cisco_aaa_model.py

```diff
@@ -49,36 +73,61 @@ class DomainStripping(str, Enum):
 class TacacsServer(FeatureTemplateValidator):
     model_config = ConfigDict(populate_by_name=True)
 
-    address: str
-    port: int = 49
-    timeout: int = Field(default=5)
-    key: str
-    secret_key: Optional[str] = Field(default=None, json_schema_extra={"vmanage_key": "secret-key"})
-    key_enum: Optional[str] = Field(default=None, json_schema_extra={"vmanage_key": "key-enum"})
+    address: str = Field(description="The IP address or hostname of the TACACS+ server")
+    port: int = Field(default=49, description="The port for the TACACS+ server")
+    timeout: int = Field(default=5, description="The timeout period in seconds for the TACACS+ server")
+    key: str = Field(description="The key for the TACACS+ server")
+    secret_key: Optional[str] = Field(
+        default=None,
+        json_schema_extra={"vmanage_key": "secret-key"},
+        description="The secret key for the TACACS+ server",
+    )
+    key_enum: Optional[str] = Field(
+        default=None,
+        json_schema_extra={"vmanage_key": "key-enum"},
+        description="The key enumeration for the TACACS+ server",
+    )
 
 
 class TacacsGroup(FeatureTemplateValidator):
     model_config = ConfigDict(populate_by_name=True)
 
-    group_name: str = Field(json_schema_extra={"vmanage_key": "group-name"})
-    vpn: int = 0
-    source_interface: Optional[str] = Field(default=None, json_schema_extra={"vmanage_key": "source-interface"})
-    server: List[TacacsServer] = []
+    group_name: str = Field(
+        json_schema_extra={"vmanage_key": "group-name"}, description="The name of the TACACS+ group"
+    )
+    vpn: int = Field(default=0, description="The VPN ID for the TACACS+ group")
+    source_interface: Optional[str] = Field(
+        default=None,
+        json_schema_extra={"vmanage_key": "source-interface"},
+        description="The source interface for the TACACS+ group",
+    )
+    server: List[TacacsServer] = Field(default=[], description="The list of TACACS+ servers for the group")
 
 
 class CiscoAAAModel(FeatureTemplate):
     model_config = ConfigDict(arbitrary_types_allowed=True, populate_by_name=True)
+    docs_description: str = Field(exclude=True, description="Cisco AAA Feature Template configuration.")
```

Comment:
I don't think that belongs to model. Use private field?
https://docs.pydantic.dev/latest/concepts/models/#private-model-attributes
```suggestion
    _docs_description: str = "Cisco AAA Feature Template configuration."
```